Database-Files/dummy.sql

```diff
@@ -0,0 +1,184 @@
+
+insert into capLead (name, photo, message)
```

Comment:
same with this script

Reply:
![image](https://user-images.githubusercontent.com/44677467/68292709-56ca4100-0084-11ea-94aa-0f305e1863a3.png)
![image](https://user-images.githubusercontent.com/44677467/68292727-5e89e580-0084-11ea-99f4-fa71a0345419.png)